test/lit.cfg

```diff
@@ -1326,6 +1326,46 @@ if platform.system() != 'Darwin':
     config.target_resilience_test = ('%s --no-symbol-diff' %
                                      config.target_resilience_test)
 
+platform_module_dir = make_path(test_resource_dir, config.target_sdk_name)
+if run_vendor != 'apple':
+    platform_module_dir = make_path(platform_module_dir, run_cpu)
+lit_config.note('Using platform module dir: ' + platform_module_dir)
+if test_sdk_overlay_dir:
+    platform_sdk_overlay_dir = test_sdk_overlay_dir
+else:
+    platform_sdk_overlay_dir = platform_module_dir
+
+# If static stdlib is present, enable static stdlib tests
+static_stdlib_path = make_path(config.swift_lib_dir, "swift_static", config.target_sdk_name)
+libswiftCore_path = make_path(static_stdlib_path, "libswiftCore.a")
+if os.path.exists(libswiftCore_path):
+    config.available_features.add("static_stdlib")
+    config.substitutions.append(('%target-static-stdlib-path', static_stdlib_path))
+    lit_config.note('using static stdlib path: %s' % static_stdlib_path)
+
+# Set up testing with the standard libraries coming from the OS / just-built libraries
+# default Swift tests to use the just-built libraries
+target_stdlib_path = platform_module_dir
+if 'use_os_stdlib' not in lit_config.params:
+	lit_config.note('Testing with the just-built libraries at ' + target_stdlib_path)
+	config.target_run = (
+        "/usr/bin/env "
+        "DYLD_LIBRARY_PATH='{0}' " # Apple option
+        "LD_LIBRARY_PATH='{0}' " # Linux option
+        .format(target_stdlib_path))
```

Comment:
This will break the Windows test suite.

Reply:
I am unfamiliar with the Windows test suite, from what I can gather the windows environment variable is PATH? Changed the PR accordingly.

Reply:
Changing PATH is not a good idea. I am gonna enable this change just on non-windows platforms for now.